spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/ContainerFactory.java

```diff
@@ -0,0 +1,16 @@
+package org.springframework.cloud.dataflow.module.deployer.kubernetes;
+
+import org.springframework.cloud.dataflow.core.ModuleDeploymentRequest;
+
+import io.fabric8.kubernetes.api.model.Container;
+
+/**
+ * Defines how a Kubernetes {@link Container} is created. 
+ * 
+ * @author Florian Rosenberg
+ */
+public interface ContainerFactory {
```

Comment:
I understand. However, this is just a simple refactoring and most of the code that was related to creating a "container" moved to the DefaultContainerFactory. We can move out the other one (`PreBaked...`) and the related configuration.